src/vm/wren_core.c

```diff
@@ -472,6 +482,11 @@ DEF_PRIMITIVE(fiber_yield1)
 
 DEF_PRIMITIVE(fn_new)
 {
+  RETURN_ERROR("Fn metaclass does not implement 'new()'.");
+}
```

Comment:
Hmm, maybe I'm missing something but the following behaviour doesn't seem correct:

```
\\/"-
 \_/   wren v0.0.0
> var g = Fn.new()
> g.call()
Segmentation fault: 11
```

It seems the default constructor is still being created for `Fn`. Now I'm not sure if the solution I have in this PR is the correct one or if maybe there's a problem with the default constructor logic in the compiler?